src/components/SingleArticle.jsx

```diff
@@ -2,17 +2,18 @@ import { useParams } from 'react-router-dom'
 import { useState, useEffect } from "react"
 import { getArticleById } from '../../api'
 import { getCommentsById } from '../../api'
-
+import { patchVotesByArticleId } from '../../api'
 
 
 export default function SingleArticle(){
     
     const {article_id} = useParams()
-    const [article, setArticle] = useState([])
+    const [article, setArticle] = useState({})
     const [loading, setLoading] = useState(true)
     const [error, setError] = useState(null)
     const [comments, setComments] = useState([])
     const [showComments, setShowComments] = useState(false)
+    const [votes, setVotes] = useState(0)
```

Comment:
I don't think you use this to render the vote count on the page. You use article.votes. this makes much more sense to me, as we do not need to duplicate stored data. I'd remove this.